### ISSUE TITLE
Add support to run as a Parent command with document.body as default value

### ISSUE THREAD

**Issue by NicholasBoll** [NicholasBoll/cypress-pipe#22]
One thing that has felt a bit off using this library is the concept of a "starter" type helper. I've always thought about Cypress test helpers having 3 kind of helpers (found [here](https://github.com/Workday/canvas-kit/blob/cb7013f4c2efe355afecd59a80276e75af4d7a88/TESTING.md#component-helpers))

They are:
* starters - start a Cypress chain with optional scoping. This becomes weird and I'll explain later.
* scoping - uses a previous subject to transform or scope. This can be used by `cypress-pipe` and will be retried
* actions - anything that performs a side-effect like `.click()`. `cypress-pipe` will not retry these, but will add before/after snapshots and the function name to the log

This makes the "starter" be in an awkward place. It has a weird signature and use. Let's say we have a series of helper functions to get an input element by its label that can be optionally scoped by a previous subject. It is common to have a login/signin from more than 1 place on the page, but you want the option to scope the function by some subject so we can test the right one. Your helper might look something like this:

```ts
const getInputByLabelText = (label: string) => (subject?: JQuery):  T extends JQuery ? JQuery : Cypress.Chainable<JQuery> => {
  // OMG, what's that `T extends JQuery`? How do I implement this in a way that makes Typescript happy?!

  // pretend we _don't_ have to keep Typrescript happy...
  if (subject) {
    return subject.find('input') // this is a jQuery object here...
  } else {
    return cy.get('input') // Cypress.Chainable
  }
}
```

```ts
// starter
getInputByLabelText('My Label')() // notice the extra (). So awkward
  .type('foo')

// scoping
cy.contains('form', 'Quick SIgn In')
  .pipe(getInputByLabelText('My Label)
  .type('foo')
```

We had to decide that this helper could _start_ a chain, or _continue_ a chain.

We could make it simpler and say _always_ returns a Cypress chain, but that's still different these helpers would have to be written differently.

If `cypress-pipe` allowed the subject to be optional and gave it `Cypress.$('body')` if the previous subject wasn't defined, we could do something like the following:

```ts
const getInputByLabelText = (label: string) => (subject: JQuery): JQuery => {
  // I'm not actually finding the input by it's label... Just return an input...
  return subject.find('input')
}
```

To use this, we can simply do:

```ts
cy.pipe(getInputByLabelText('My Label'))
  .type('foo')
```

The signature of `getInputByLabelText` no longer needs anything special. It just assumes a jQuery subject is always there. `cypress-pipe` can still throw Typescript errors if the subject is incorrect for the function passed to it.

This gets rid of the awkward starter problem.

**Comment by NicholasBoll** [comment]
@pnaza @alindber

**Comment by NicholasBoll** [comment]
Released as 1.7.0
